commonutils/src/main/java/io/intrepid/commonutils/StringUtils.java

```diff
@@ -0,0 +1,504 @@
+package io.intrepid.commonutils;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.text.Editable;
+import android.text.SpannableStringBuilder;
+import android.text.Spanned;
+import android.text.TextUtils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.StringWriter;
+import java.io.UnsupportedEncodingException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * A utility class bundling a set of handy to use methods that deal with {@link String},
+ * {@link CharSequence}, {@link android.text.Spannable}, etc.
+ * <p/>
+ * Created by Anton Spaans on 4/15/16.
+ */
+@SuppressWarnings("unused")
+public abstract class StringUtils {
```

Comment:
Why? That is just too wordy. The package name already makes sure the name is scoped properly.